### PR TITLE
DWPF-703 removes email sending logic, notify package and  its dependencies

### PR DESCRIPTION
**Pull request by nicopicchio**
removed logic inside form_valid()
removed code inside notify.py
removed notifications-python-client with poetry remove

### CODE REVIEW

File: django_feedback_govuk/views.py [hunk 1]
```diff
@@ -98,7 +87,7 @@ def dispatch(
             return redirect(
                 reverse(
                     "feedback-listing",
-                    kwargs={
+                    kwargs = {
```
Comment (CamLamb) [inline django_feedback_govuk/views.py:90]: **Code Style:**
Calling a function using keywords should have no spaces around the `=`

File: django_feedback_govuk/views.py [hunk 2]
```diff
@@ -137,7 +126,7 @@ def get_context_data(self, **kwargs: Any) -> Dict[str, Any]:
         context = super().get_context_data(**kwargs)
 
         form = self.feedback_form(
-            initial={
+            initial = {
```
Comment (CamLamb) [inline django_feedback_govuk/views.py:129]: Same here

File: django_feedback_govuk/views.py [hunk 3]
```diff
@@ -147,11 +136,11 @@ def get_context_data(self, **kwargs: Any) -> Dict[str, Any]:
...
-            form_id=self.form_id,
-            form=form,
-            fields=form_fields,
-            model=self.feedback_model,
-            hide_back_button=len(dfg_settings.FEEDBACK_FORMS) == 1,
+            form_id = self.form_id,
+            form = form,
+            fields = form_fields,
+            model = self.feedback_model,
+            hide_back_button = len(dfg_settings.FEEDBACK_FORMS) == 1,
```
Comment (CamLamb) [inline django_feedback_govuk/views.py:143]: And again, was this manually done or are you using a code formatter?

**Review by CamLamb — COMMENTED** [review]
LGTM!

The next step is to add documentation to the package's `README.md` to give guidance to users of the package on how they could write a script to send a notification email (see the example in the ticket)

File: django_feedback_govuk/views.py (hunk 3 above)
Comment (nicopicchio) [reply django_feedback_govuk/views.py:143]: Not manually done, so it must happen on save? Surely poetry does not do that

File: README.md [hunk 4]
```diff
@@ -240,3 +235,19 @@ poetry config pypi-token.pypi XXXXXXXX
 ```
 
 Now the make commands should work as expected.
+
+## Sending automated emails after form submission
+
+Based on the needs of your Service you may want the package to inform you via email of the feedback received. For example you might want to receive an email if there is new feedback to be reviewed and call the method on a regular schedule (e.g. once a day) through cron, Celery Beat, etc.
```
Comment (CamLamb) [inline README.md:241]:
```suggestion
Based on the needs of your project you may want to be notified of the feedback received. For example you might want to receive an email if there is new feedback to be reviewed. To do this you can create a method and call it on a regular schedule (e.g. once a day) through cron, Celery Beat, etc.
```

File: django_feedback_govuk/templates/django_feedback_govuk/widgets/star_rating/star_rating.html
[the remark this answers is not in the document]
Comment (nicopicchio) [reply django_feedback_govuk/templates/django_feedback_govuk/widgets/star_rating/star_rating.html:1]: I have not checked it, I only ran `make lint` as it was complaining for something when I ran `make check`

File: django_feedback_govuk/templates/django_feedback_govuk/widgets/star_rating/radios.html
[the remark this answers is not in the document]
Comment (nicopicchio) [reply django_feedback_govuk/templates/django_feedback_govuk/widgets/star_rating/radios.html:1]: star rating was tested and working as expected.

**Review by CamLamb — APPROVED** [review]
LGTM

**Pull request merged**